### ISSUE TITLE
Separate 10.0.0.0/19 IPMI network from VL 930

### ISSUE THREAD

**Issue by hakasapl** [CCI-MOC/ops-issues#1273]
It was initially looped in with VL 930 during the great Cisco switch migration of 2023, it will need to be separated into its own VLAN.

* A new vlan will need to be created in the ansible switches vlan file
* OVirt will need a new vlan defined, that vlan should be tagged on the TORS switchport

**Comment by naved001** [comment]
@hakasapl I don't think we have much left on this network (10.0.0.0/19), so we could close this issue. Just confirming with you before closing.

**Comment by hakasapl** [comment]
Sounds good
